app/Rules/MatchPayloadParams.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Rules;
+
+use ErrorException;
+use Illuminate\Contracts\Validation\Rule;
+use Throwable;
+
+class MatchPayloadParams implements Rule
+{
+    private $errors;
+    private $payloadParams;
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct($payloadParams)
+    {
+        $this->payloadParams = $payloadParams;
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $payload = json_decode($this->payloadParams);
+        for ($i = 0; $i < count($value); $i++) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Rules/MatchPayloadParams.php</br>**Line**: 35</br>String "return " does not require double quotes; use single quotes instead